openapi/components/schemas/users/Plan.yaml

```diff
@@ -45,6 +45,12 @@ properties:
     items:
       type: array
       items: {}
+    example:
+      - - ACTORS
```

Comment:
Why the double `- -`?

Reply:
It was like that in the original example 😅